src/main/java/mate/academy/spring/model/dto/response/ShoppingCartResponseDto.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.spring.model.dto.response;
+
+import java.util.List;
+import lombok.Data;
+import mate.academy.spring.model.Ticket;
+
+@Data
+public class ShoppingCartResponseDto {
+    private Long id;
+    private List<Ticket> tickets;
```

Comment:
Lets return List of ticket ids here. (If you want return more information about tickets, you should create some TicketResponseDto, and return List<TicketResponseDto> here).

Reply:
done